src/app/rundown-view/components/hover-scrub/hover-scrub/hover-scrub.component.ts

```diff
@@ -0,0 +1,136 @@
+import { ConfigurationService } from 'src/app/shared/services/configuration.service'
+
+export interface VideoHoverScrubPositonsAndMoment {
+  whereIsUserCursorInPercent: number
+  top: number
+  left: number
+  isShown: boolean
+  playedDurationForPartInMs?: number
+}
+
+export interface HoverScrubElementSizes {
+  width: number
+  height: number
+}
+
+@Component({
+  selector: 'sofie-hover-scrub',
+  templateUrl: './hover-scrub.component.html',
+  styleUrls: ['./hover-scrub.component.scss'],
+})
+export class HoverScrubComponent implements OnInit, OnDestroy {
+  @Input() public playedDurationForPartInMs: number
+  @Input() public type: string
+  @Input() public fileName?: string
+  public hoverScrubVideoSrc: string
+  private readonly hoverScrubTopOffset = 15
+  private readonly timeoutDurationAfterMouseMoveInMs = 5
+  private studioConfiguration: StudioConfiguration
+  private configurationServiceSubscription: Subscription
+  private timeoutAfterMouseMove?: NodeJS.Timeout
+  public videoHoverScrubPositonsAndMoment: VideoHoverScrubPositonsAndMoment
+  public hoverScrubElementSizes: HoverScrubElementSizes
+  @ViewChild('hoverScrubElement')
+  public hoverScrubElement: ElementRef<HTMLDivElement>
+  public hoverScrubTooltipElementRef: HTMLDivElement
+
+  constructor(private readonly configurationService: ConfigurationService) {}
+
+  public get isVideoHoverScrub(): boolean {
+    return this.type === Tv2PieceType.VIDEO_CLIP
+  }
+
+  public onMouseEnterInVideoTimeline(event: MouseEvent): void {
+    this.calculateIsUserCursorInPercent(event)
+  }
+
+  public onMouseMoveInVideoTimeline(event: MouseEvent): void {
+    this.calculateIsUserCursorInPercent(event)
+  }
+
+  public onMouseLeaveInVideoTimeline(): void {
+    this.videoHoverScrubPositonsAndMoment = {
+      ...this.videoHoverScrubPositonsAndMoment,
+      isShown: false,
+    }
+  }
+
+  public ngOnInit(): void {
+    this.setWidthAndHeightBasedOnType()
+    this.appendHoverScrubTooltipElementToBody()
+    if (this.isVideoHoverScrub) {
+      this.configurationServiceSubscription = this.configurationService.getStudioConfiguration().subscribe((studioConfiguration: StudioConfiguration) => {
+        this.studioConfiguration = studioConfiguration
+        this.setVideoSrc()
+      })
+    }
+  }
+
+  private appendHoverScrubTooltipElementToBody(): void {
+    this.hoverScrubTooltipElementRef = document.createElement('div')
+    this.hoverScrubTooltipElementRef.className = 'c-sofie-hover-scrub-tooltip'
+    const body: HTMLElement = document.getElementsByTagName('body')[0]
+    body.append(this.hoverScrubTooltipElementRef)
+  }
+
+  public ngOnDestroy(): void {
+    this.configurationServiceSubscription?.unsubscribe()
+    this.hoverScrubTooltipElementRef.remove()
+  }
+
+  private setWidthAndHeightBasedOnType(): void {
+    this.hoverScrubElementSizes = {
+      width: this.type === Tv2PieceType.VIDEO_CLIP ? 170 : 200,
+      height: this.type === Tv2PieceType.VIDEO_CLIP ? 120 : 60,
```

Comment:
We need to know the size of this container, because we make in with absolute position. I make it constants in px. I looking forward  if we find better solution.